tests/main/interfaces-desktop-host-fonts/task.yaml

```diff
@@ -16,12 +16,7 @@ restore: |
     rm -f /usr/lib/fontconfig/cache/cache.txt
 
 execute: |
-    #shellcheck source=tests/lib/pkgdb.sh
-    . "$TESTSLIB"/pkgdb.sh
-    distro_install_package fontconfig
-
```

Comment:
I'm curious about this part. There are changes with additions of fontconfig above and here removal of the installation of the fontconfig package. What is this about?

Reply:
Ahh, I'm blind. Sorry :)